### PR TITLE
Added sideness check in SleepingHandler.trySleep().

### PR DESCRIPTION
**Pull request by stfwi**
Hi Vazkii, minimal code change sidenes check added in SleepingHelper, came across this from this issue here: https://github.com/stfwi/engineers-tools/issues/8.
It's probably faster for you to quickly add the line yourself and close the PR ;) 
Cheers, wile -

### CODE REVIEW

File: src/main/java/vazkii/botania/common/core/handler/SleepingHandler.java [hunk 1]
```diff
@@ -11,18 +11,20 @@
...
+          .filter(e -> e instanceof EntityDoppleganger)
+          .anyMatch(e -> ((EntityDoppleganger) e).getPlayersAround().contains(event.getPlayer()));
 
-		if(nearGuardian) {
-			event.setResult(PlayerEntity.SleepResult.NOT_SAFE);
-		}
+      if(nearGuardian) {
+        event.setResult(PlayerEntity.SleepResult.NOT_SAFE);
+      }
+    }
```
Comment (Hubry) [inline src/main/java/vazkii/botania/common/core/handler/SleepingHandler.java]: Don't change formatting to spaces

**Comment by Hubry** [comment]
Do you actually need to check the sleep result on the client? Vanilla calls this only on the server. We'll probably pull anyways, though.
BTW your 1.14 sources don't seem to be up on GitHub? I wanted to check what exactly are you doing there.

File: src/main/java/vazkii/botania/common/core/handler/SleepingHandler.java (hunk 1 above)
Comment (stfwi) [reply src/main/java/vazkii/botania/common/core/handler/SleepingHandler.java]: Hmm my IDE normally adapts to indent in the file ... thanks for the hint.

**Comment by stfwi** [comment]
You mean from the tools mod? These are here: https://github.com/stfwi/engineers-tools/blob/develop/1.14/src/main/java/wile/engineerstools/items/ItemSleepingBag.java.

Is there any action needed in this PR thread?

**Comment by stfwi** [comment]
I forgot, yes I would check client side, the exception that was shown in the log passed to me was 
```
java.lang.ClassCastException: net.minecraft.client.world.ClientWorld cannot be cast to net.minecraft.world.server.ServerWorld
	at vazkii.botania.common.core.handler.SleepingHandler.trySleep(SleepingHandler.java:20) ~[?:r1.11-375] {re:classloading}
```

**Pull request merged**